src/components/form/form.stories.mdx

```diff
@@ -0,0 +1,118 @@
+
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+import Form from '.';
+import Textbox from '../../__experimental__/components/textbox';
+import Button from '../button';
+
+<Meta
+  title="Design System/Form"
+  parameters={{ info: { disable: true }}}
+/>
+
+# Form
+Represents a document section containing interactive controls for submitting information.
+
+## Quick Start
+
+To use Form, import the `Form` and pass the content, usually various inputs, as children.
+
+Save button is meant to be passed as `saveButton` prop - 
```

Comment:
I think this is too detailed.
```diff
- Save button is meant to be passed as `saveButton` prop - 
- it will be wrapped internally by validation summary if `showSummary` prop is passed as `true`.

+ You can provide a save button using the `saveButton` prop.
+ You can hide the validation summary by setting the `showSummary` prop to false.
```